docs/api.md

```diff
@@ -994,6 +996,9 @@ puppeteer.launch().then(async browser => {
 });
 ```
 
+> **NOTE** Request interception doesn't work with data URLs. Calling `abort`,
+> `continue` or `respond` on requests for data URLs is a noop.
```

Comment:
no-op

Reply:
Found "noop" in google, it's apparently a thing as well :)